conf/BUILD

```diff
@@ -0,0 +1,60 @@
+files(
+    name="st2client_sample_config",
+    sources=["st2rc.sample.ini"],
+)
+
```

Comment:
I don't understand the difference between file and files here?
Can you add a comment as unclear why file used in some cases and files in others, when the source seems to be a singleton in both cases?

Reply:
Because I started with the plural `files` form everywhere, and then switched some of them to the singular `file`.

If it's likely we will add more files later, then I would want to keep the plural form. But here, a simple `file` should work just fine. I'll review and switch some more of them.

Reply:
OK. I rebased on master and updated the commits to prefer the singular where possible.